configs/community.dbuild

```diff
@@ -1007,62 +1004,74 @@ build += {
-    extra.projects: ["scalametaJVM"]  // no Scala.js
+    extra.projects: ["scalametaJVM", "contribJVM", "semanticdbScalac"]  // no Scala.js
     extra.exclude: [
-      // requires scalatex-site which requires Scala.js
-      "readme"
       // we never build sbt plugins or benchmarks
-      "scalahostSbt", "benchmarks"
-      // test failures it doesn't seem worth investigating; pulls down external source trees
-      "contribJVM"
+      "benchmarks"
     ]
     // use right version-specific source directories regardless of our weird dbuild Scala version numbers
     extra.commands: ${vars.default-commands} [
       "set unmanagedSourceDirectories in (commonJVM, Compile) += (baseDirectory in commonJVM).value / \"src\" / \"main\" / \"scala-2.12\""
       "set unmanagedSourceDirectories in (dialectsJVM, Compile) += (baseDirectory in dialectsJVM).value / \"src\" / \"main\" / \"scala-2.12\""
       "set unmanagedSourceDirectories in (scalametaJVM, Compile) += (baseDirectory in scalametaJVM).value / \"src\" / \"main\" / \"scala-2.12\""
-      "set unmanagedSourceDirectories in (scalahost, Compile) += (baseDirectory in scalahost).value / \"src\" / \"main\" / \"scala-2.12.1\""
+      "set unmanagedSourceDirectories in (semanticdbScalac, Compile) += (baseDirectory in semanticdbScalac).value / \"src\" / \"main\" / \"scala-2.12.3\""
     ]
   }
 
-  // forked (February 2017) to remove wartremover and coursier
-  // fork tweaked March 2017 to alter scalameta dependency
-  // caution, refreshing the fork any further could be problematic;
-  // see Olafur's comment at
+  // forked (refreshed September 2017) to remove coursier
+  // caution, refreshing the fork any further can be problematic;
+  // see Olafur's remarks at
   // https://github.com/scala/community-builds/issues/499#issuecomment-287307613
   // about the instability of both scalameta and scalafix
   ${vars.base} {
     name: "scalafix"
     uri:  ${vars.uris.scalafix-uri}
     extra.exclude: [
       "scalafix-sbt"  // we never build sbt plugins
-      // "These tests depend on various projects (circe, shapeless,...) to
-      // reproduce bugs that surfaced in larger integration tests"
-      // says Olaf.  depends on circe which we don't have here yet
-      "scalafix-nsc"
-      // didn't work, and anyway, Olafur says it isn't appropriate to include;
-      // it "involves cloning other projects and compiling them"
-      "scalafix-tests"
       // Missing dependency: the library com.github.alexarchambault#case-app
       "cli"
       // requires scalatex-site which requires Scala.js
       "readme"
+      // no Scala.js please
+      "coreJS", "diffJS"
+      // Missing dependency: the library org.scalameta#semanticdb-sbt
+      "testsInputSbt"
+      // do I *look* like Dotty?
+      "testsOutputDotty"
+      // can't expand macros compiled by previous versions of Scala
+      "testsOutputSbt"
+      // Missing dependency: the library org.scalameta#semanticdb-sbt-runtime is not provided
```

Comment:
This is a small library in https://github.com/scalameta/semanticdb-sbt the `unit` project is where most of the interesting scalafix unit tests exist.